quick_start/kusto_sample_config.json

```diff
@@ -1,30 +1,33 @@
-	"ingestUri" : "https://ingest-sdkse2etest.eastus.kusto.windows.net",
-	"databaseName" : "e2e",
-	"tableName" : "SampleTable",
-	"useExistingTable": false,
-	"alterTable": true,
-	"queryData": true,
-	"ingestData": true,
-	"tableSchema" : "(rownumber:int, rowguid:string, xdouble:real, xfloat:real, xbool:bool, xint16:int, xint32:int, xint64:long, xuint8:long, xuint16:long, xuint32:long, xuint64:long, xdate:datetime, xsmalltext:string, xtext:string, xnumberAsText:string, xtime:timespan, xtextWithNulls:string, xdynamicWithNulls:dynamic)",
-	"data" :
-	[
-		{
-			"sourceType": "localFileSource",
-			"dataSourceUri": "dataset.csv",
-			"format": "CSV",
-			"useExistingMapping": true,
-			"mappingName": "",
-			"mappingValue": ""
-		},
-		{
-			"sourceType": "localFileSource",
-			"dataSourceUri": "dataset.json",
-			"format": "MULTIJSON",
-			"useExistingMapping": false,
-			"mappingName": "SampleTableMapping",
-			"mappingValue": "[{\"Properties\":{\"Path\":\"$.rownumber\"},\"column\":\"rownumber\",\"datatype\":\"int\"}, {\"Properties\":{\"Path\":\"$.rowguid\"},\"column\":\"rowguid\",\"datatype\":\"string\"}, {\"Properties\":{\"Path\":\"$.xdouble\"},\"column\":\"xdouble\",\"datatype\":\"real\"}, {\"Properties\":{\"Path\":\"$.xfloat\"},\"column\":\"xfloat\",\"datatype\":\"real\"}, {\"Properties\":{\"Path\":\"$.xbool\"},\"column\":\"xbool\",\"datatype\":\"bool\"}, {\"Properties\":{\"Path\":\"$.xint16\"},\"column\":\"xint16\",\"datatype\":\"int\"}, {\"Properties\":{\"Path\":\"$.xint32\"},\"column\":\"xint32\",\"datatype\":\"int\"}, {\"Properties\":{\"Path\":\"$.xint64\"},\"column\":\"xint64\",\"datatype\":\"long\"}, {\"Properties\":{\"Path\":\"$.xuint8\"},\"column\":\"xuint8\",\"datatype\":\"long\"}, {\"Properties\":{\"Path\":\"$.xuint16\"},\"column\":\"xuint16\",\"datatype\":\"long\"}, {\"Properties\":{\"Path\":\"$.xuint32\"},\"column\":\"xuint32\",\"datatype\":\"long\"}, {\"Properties\":{\"Path\":\"$.xuint64\"},\"column\":\"xuint64\",\"datatype\":\"long\"}, {\"Properties\":{\"Path\":\"$.xdate\"},\"column\":\"xdate\",\"datatype\":\"datetime\"}, {\"Properties\":{\"Path\":\"$.xsmalltext\"},\"column\":\"xsmalltext\",\"datatype\":\"string\"}, {\"Properties\":{\"Path\":\"$.xtext\"},\"column\":\"xtext\",\"datatype\":\"string\"}, {\"Properties\":{\"Path\":\"$.rowguid\"},\"column\":\"xnumberAsText\",\"datatype\":\"string\"}, {\"Properties\":{\"Path\":\"$.xtime\"},\"column\":\"xtime\",\"datatype\":\"timespan\"}, {\"Properties\":{\"Path\":\"$.xtextWithNulls\"},\"column\":\"xtextWithNulls\",\"datatype\":\"string\"}, {\"Properties\":{\"Path\":\"$.xdynamicWithNulls\"},\"column\":\"xdynamicWithNulls\",\"datatype\":\"dynamic\"}]"
-		}
-	]
+  "kustoUri": "https://help.kusto.windows.net",
+  "ingestUri": "https://ingest-help.kusto.windows.net",
+  "databaseName": "MyDatabase",
+  "tableName": "SampleTable",
+  "useExistingTable": true,
+  "alterTable": true,
+  "queryData": true,
+  "ingestData": true,
+  "AuthenticationMode": "UserPrompt",

```

Comment:
Some properties are PascalCase and some are camelCase, was this also that way in c#?
That's a bit weird.

Reply:
True, it was due moving some of the class vars to the JSON. the same goes for C# and node. ill change it all to camelCase.